### PR TITLE
[#11335] Use Pokt network as an infura replacement

### LINKED ISSUE #11335: [Spike] Use Pokt network as an infura replacement
As a developer
I want to use Pokt instead of Infura 
As I want to use decentralized technologies

### Description

Assess the viability of using https://www.pokt.network/ as an infura replacement

We should start from using their gateway, and making sure that the app works as expected, later, we can move to https://github.com/status-im/status-react/issues/13591

### PR DESCRIPTION
**Pull request by rasom**
fix #11335

status: ready

### CODE REVIEW

**Comment by jakubgs** [comment]
This will also need change in several places in `status-jenkins-lib`:
https://github.com/status-im/status-jenkins-lib/blob/88a0fd20080b9984fb373c5232d9bdb03d67603c/vars/ios.groovy#L10-L11
https://github.com/status-im/status-jenkins-lib/blob/88a0fd20080b9984fb373c5232d9bdb03d67603c/vars/android.groovy#L21-L22
https://github.com/status-im/status-jenkins-lib/blob/88a0fd20080b9984fb373c5232d9bdb03d67603c/vars/creds.groovy#L13-L23
Also ones for desktop app. That is assuming we'll have separate token for releases and e2e builds as well.

**Comment by jakubgs** [comment]
I created the Devel and E2E apps in POKT and added the tokens/IDs to Jenkins:

image

**Comment by pavloburykh** [comment]
@rasom hi! Regarding failed tests:

1. [test_sticker_1_1_public_chat_mainnet](https://ethstatus.testrail.net/index.php?/tests/view/3020483), id: 702157 is reproducible in PR build. 

### ISSUE 1 Network URL is invalid when switching to Mainnet

**Steps:**
1. Login multiaccount on PR build
2. Go to Profile - Advanced - Network
3. Try to switch to Mainnet
4. Observe an error

**Actual result:** Network URL is invalid when switching to Mainnet

2. [test_wallet_tx_history_copy_tx_hash_on_cellular](https://ethstatus.testrail.net/index.php?/tests/view/3020477), id: 700756 is reproducible in PR build. 

### ISSUE 2 Endless loader on History tab if transaction history is empty

Steps: 

**Steps:**
1. Login multiaccount on PR build
2. Open Wallet that does not contain any history of transactions
3. Open history tab
4. Observe the result

**Actual result:** Endless loader

**Expected result:** "No transactions in your history yet" info is displayed

[test_1_1_chat_text_message_edit_delete_push_disappear](https://ethstatus.testrail.net/index.php?/tests/view/3020475), id: 695843 and [test_public_chat_message_edit](https://ethstatus.testrail.net/index.php?/tests/view/3020476), id: 700734 are also reproducible in PR but are not PR related as I am also reproducing those bugs in another PR. Bugs are likely introduced  by new go version. I've mentioned about this issue here https://discord.com/channels/624205794384281629/626738033701683200/1030101011085398037


Need some more time to figure out these ones:
[test_keycard_send_tx_sign_message_request_stt_testdapp](https://ethstatus.testrail.net/index.php?/tests/view/3020482), id: 700769
[test_send_tx_set_recipient_options](https://ethstatus.testrail.net/index.php?/tests/view/3020478), id: 700757

**Comment by rasom** [comment]
ISSUE 2 is fixed

**Review by Parveshdhull — APPROVED** [review]

File: src/status_im/utils/config.cljs [hunk 1]
```diff
@@ -18,12 +18,16 @@
...
+(goog-define POKT_TOKEN "3ef2018191814b7e1009b8d9")
 (goog-define OPENSEA_API_KEY "")
 
-(def mainnet-rpc-url (str "https://mainnet.infura.io/v3/" INFURA_TOKEN))
-(def testnet-rpc-url (str "https://ropsten.infura.io/v3/" INFURA_TOKEN))
-(def goerli-rpc-url  (str "https://goerli.infura.io/v3/" INFURA_TOKEN))
+(def mainnet-rpc-url
+  (str "https://eth-archival.gateway.pokt.network/v1/lb/" POKT_TOKEN))
+
+(def testnet-rpc-url
```
Comment (flexsurfer) [inline src/status_im/utils/config.cljs:27]: ropsten is still needed?

Comment (rasom) [reply src/status_im/utils/config.cljs:27]: @flexsurfer no but I will remove in a separate PR anyway

Comment (flexsurfer) [reply src/status_im/utils/config.cljs:27]: ok

**Pull request merged**